### PR TITLE
Implement ocean parameters, additional filtering, and run model 15,000 times

### PR DESCRIPTION
**Pull request by leeyap**
Title says it all.

Major changes:
- Four new ocean parameters
- Forced joint PDF removed
- Test with new filtering methods and additional constraints
- Run 15,000 times, adjust scripts and outputs accordingly

### CODE REVIEW

File: workflow/01_generate_data.Rmd [hunk 1]
```diff
@@ -114,60 +130,6 @@ param_sce <- generate_params(runlist$run_number)
 runlist <- left_join(runlist, param_sce, by = "run_number")
 ```
 
-``` {r joint-params, fig.cap = " The final joint distribution of ECS and diffusivity. The individual parameter distributions are shown on the x and y axes."}
```
Comment (bpbond) [inline workflow/01_generate_data.Rmd:117]: Too bad we're deleting this—you did great work here—but hurray to simplifying.

**Review by bpbond — DISMISSED** [review]
Looks great! I pushed a change to `./github/workflows/render-markdown.yaml` to try to fix the GitHub Actions check.

**Comment by bpbond** [comment]
@leeyap Is it correct that this PR is against `main`? Or should it be to `reviews`?

**Comment by bpbond** [comment]
>All checks have failed

Hmm @leeyap could you confirm that `03_generate_figures.Rmd` runs OK in a clean R session?

```
Quitting from lines 240-320 (03_generate_figures.Rmd) 
[849](https://github.com/JGCRI/trackingC/actions/runs/4015948621/jobs/6898320496#step:7:850)
Error in `mutate()`:
[850](https://github.com/JGCRI/trackingC/actions/runs/4015948621/jobs/6898320496#step:7:851)
! Problem while computing `param = factor(...)`.
[851](https://github.com/JGCRI/trackingC/actions/runs/4015948621/jobs/6898320496#step:7:852)
Caused by error in `factor()`:
[852](https://github.com/JGCRI/trackingC/actions/runs/4015948621/jobs/6898320496#step:7:853)
! object 'param' not found
[853](https://github.com/JGCRI/trackingC/actions/runs/4015948621/jobs/6898320496#step:7:854)
Backtrace:
[854](https://github.com/JGCRI/trackingC/actions/runs/4015948621/jobs/6898320496#step:7:855)
 1. ... %>% ggplot(aes(year, value, fill = param))
 2. ```

**Comment by leeyap** [comment]
It's working for me, hmm. Let me investigate

**Pull request merged**